app/src/main/java/com/duckduckgo/app/browser/httpauth/db/WebViewStoreExtensions.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2020 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser.httpauth.db
+
+import android.os.Build
+import android.webkit.WebView
+import android.webkit.WebViewDatabase
+
+fun WebView.clearFormDataCompat() {
+    clearFormData()
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
+        val webViewDatabase = WebViewDatabase.getInstance(this.context)
+        webViewDatabase.clearFormData()
```

Comment:
NIT: could we suppress this warning since we already have the API version guard?